Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Compositor/Calibration.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using System.IO;
+using System;
+using System.Text;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.Compositor
+{
+    public class Calibration : MonoBehaviour
```

Comment:
>MonoBehaviour [](start = 31, length = 13)

it seems like we may not need this to be a monobehaviour. I may have missed something, but it seems like this class is only used by the compositionmanager.

Reply:
Calibration used to be a Singleton, but I've refactored this so that the class shouldn't be needed anymore, instead an ICalibrationData will be a required parameter to EnableHolographicCamera